### ISSUE TITLE
Exception if url is static/.env

### ISSUE THREAD

**Issue by giovannigiordano** [zeit/next.js#5775]
# Bug report
## Describe the bug
Hitting the url `static/.env` throws an exception and doesn't render the error component
```
{ NotFoundError: Not Found
    at SendStream.error (/Users/giovannigiordano/Develop/www2/node_modules/next/node_modules/send/index.js:270:31)
    at SendStream.pipe (/Users/giovannigiordano/Develop/www2/node_modules/next/node_modules/send/index.js:585:14)
    at /Users/giovannigiordano/Develop/www2/node_modules/next/dist/server/render.js:516:28
    at new Promise (<anonymous>)
    at new F (/Users/giovannigiordano/Develop/www2/node_modules/core-js/library/modules/_export.js:36:28)
    at serveStatic (/Users/giovannigiordano/Develop/www2/node_modules/next/dist/server/render.js:510:10)
    at Server._callee15$ (/Users/giovannigiordano/Develop/www2/node_modules/next/dist/server/next-server.js:704:49)
    at tryCatch (/Users/giovannigiordano/Develop/www2/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/giovannigiordano/Develop/www2/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/giovannigiordano/Develop/www2/node_modules/regenerator-runtime/runtime.js:114:21) message: 'Not Found', expose: false }
```

## To Reproduce
1. Go to `${your-app-url}/static/.env`
2. See  internal server error, not rendering _error.js

## Expected behavior
At least render the `_error.js` component

## System information
 - OS: macOS
 - Browser: chrome
 - Version of Next.js: 7.0.2

## Additional context
It's not useful, it's a nonsense that a user tries to visit this url

**Comment by timneutkens** [comment]
What is the exception?

**Comment by timneutkens** [comment]
Fixed in #5779
